src/app/containers/ATIAnalytics/index.test.jsx

```diff
@@ -6,6 +6,8 @@ import { articleDataNews } from '#pages/ArticlePage/fixtureData';
 import mapAssetData from '#pages/MediaAssetPage/fixtureData.json';
 import pglAssetData from '#pages/PhotoGalleryPage/fixtureData.json';
 import styAssetData from '#pages/StoryPage/fixtureData/mundo.json';
+import styUkrainianAssetData from '#data/ukrainian/cpsAssets/news-53561143.json';
+import styUkrainianInRussianAssetData from '#data/ukrainian/cpsAssets/features-russian-53477115.json';
```

Comment:
A number of UK and UK-in-RU CPS assets were added prior to this PR – did we/can we check they can't be used before adding these?